src/NuGet.Clients/NuGet.CommandLine/Commands/ProjectFactory.cs

```diff
@@ -86,6 +87,50 @@ public ProjectFactory(string msbuildDirectory, string path, IDictionary<string,
             Initialize(project);
         }
 
+        // This handler is called only when the common language runtime tries to bind to the assembly and fails
+        private Assembly AssemblyResolve(object sender, ResolveEventArgs args)
+        {
+            if (string.IsNullOrEmpty(_msbuildDirectory))
+            {
+                return null;
+            }
+
+            var failingAssemblyFilename = args.Name.Substring(0, args.Name.IndexOf(","));
+
+            // If we're failing to load a resource assembly, we need to find it in the appropriate subdir
+            if (failingAssemblyFilename.Length > 10 &&
+                failingAssemblyFilename.Substring(failingAssemblyFilename.Length - 10, 10).Equals(".resources", StringComparison.OrdinalIgnoreCase))
+            {
+                var fallBackToEnglish = false;
+                var cultureName = CultureInfo.CurrentCulture?.TwoLetterISOLanguageName;
```

Comment:
I don't think the ``?`` is needed here, this doesn't seem like it can be null
http://stackoverflow.com/questions/13359141/can-cultureinfo-currentculture-ever-be-null

Doesn't really change anything, it just kind of implies to a human reader that more could happen here with this operator.